python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -770,11 +774,13 @@ export default function AIConfigEditor({
         // Keep this here in case any server implementations don't return
         // aiconfig as a streaming format
         if (serverConfigResponse?.aiconfig) {
+          // Do we need to log here
```

Comment:
Yes, this is for external implementations that might not have streaming implemented for the run callback

Reply:
added and removed comment